internal/searcher/model.go

```diff
@@ -29,3 +29,28 @@ type SearchResult struct {
 	Results     []Result `json:"results"`
 	Suggestions []Result `json:"suggestions"`
 }
+
+// Package api:
+// https://search.devbox.sh/pkg/<name>
```

Comment:
Yeah, I think it would be a good idea to allow lookup by ID. The information needed to unambiguously identify a package is name, version, and system.

The `/pkg/<name>` endpoint will get you the correct information, but the client does have to iterate through the results to find the right name + version + system combination itself, which isn't ideal.

Reply:
Discussed with @gcurtis 

1. He'll add the required information to the old search api endpoint. That way, we'll get the extra info this `/pkg` api provides during the same `devbox add` call.
2. For now, I can use the new endpoint to continue working on this.